packages/integration-karma/test/shadow-dom/event-in-shadow-tree/propagation.spec.js

```diff
@@ -321,6 +325,75 @@ describe('event propagation', () => {
 
             expect(actualLogs).toEqual(expectedLogs);
         });
+
+        describe('when non-composed event crosses immediate shadow root boundary', () => {
```

Comment:
This description is a little vague for me. I think it's ok to be super clear about what we're testing.
```suggestion
        describe('when the ENABLE_NON_COMPOSED_EVENTS_LEAKAGE flag is enabled', () => {
```